classes/Language.php

```diff
@@ -118,33 +128,33 @@ public function getFields()
     /**
      * Generate translations files.
      */
-    protected function _generateFiles($newIso = null)
+    protected function generateFiles($newIso = null)
```

Comment:
Please alias `_generateFiles` method  to `generateFiles` in order to preserve backward compatibility.

ping @julienbourdeau